raster-foundry/raster-foundry
# [WIP] Add SceneToLayerDao

## Overview

WIP. More to come once all done. Branched out from https://github.com/raster-foundry/raster-foundry/pull/4479 -- will rebased once merged. Test will fail or not, idk.

### Checklist


### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.


## Testing Instructions

 * How to test this PR
 * Prefer bulleted description
 * Start after checking out this branch
 * Include any setup required, such as bundling scripts, restarting services, etc.
 * Include test case, and expected output

Closes https://github.com/raster-foundry/raster-foundry/issues/4484
